src/bci_build/package/templates/access_protected_images.j2

```diff
@@ -0,0 +1,37 @@
+## Accessing the Container Image
+
+This Container Image requires an subscription from SUSE to access. In order
+to access the container image, you must login to the SUSE Registry with your credentials. There are three ways to do that.
+
+#### Use the system registration of your host system
+
+If the host system you are using to build or run a container is already registered with the correct subscription required for accessing the LTSS container images, you can use the registration information from the host to log in to the registry.
```

Comment:
Sounds like German sentence order to me, afaik the sentences should be the other way around.

Reply:
this is the official text from the current documentation. I'm not changing it in this PR. lets do this in a followup where documentation team can weigh in on the changes.